lib/katello/engine.rb

```diff
@@ -229,7 +229,9 @@ def find_assets(args = {})
         api_view :list => 'katello/api/v2/subscription_facet/base_with_root', :single => 'katello/api/v2/subscription_facet/show'
       end
 
-      ::SettingsHelper.send :include, Katello::Concerns::SettingsHelperExtensions
+      ::SettingsController.class_eval do
+        helper Katello::Concerns::SettingsHelperExtensions
```

Comment:
If it's now going to be treated as a top-level helper it shouldn't be an ActiveConcern anymore.

Seems odd this fixes anything though, what was wrong w/ the SettingsHelper concern include? Plenty of plugins extend helpers.